api.bs

```diff
@@ -1967,6 +1967,32 @@ this difference cannot be detected
 by the site receiving the conversion report.
 
 
+## Unconfigured Browsers ## {#unconfigured}
+
+This API assumes that [=user agent=] instance
+have all of the configuration necessary
+to answer requests to <a for=PrivateAttribution>measureConversion()</a>.
+
+If the [=aggregation service=] configuration necessary
+for invoking <a for=PrivateAttribution>measureConversion()</a>
+is either absent or out of date,
+that could be observable to sites.
+Detecting stale or absent configuration might be used
+to [=fingerprint=] the [=user agent=].
+
+A [=user agent=] <span class=allow-2119>should</span> therefore
+prioritize the acquisition of [=aggregation service=] configuration
+in a way that is not observable to sites.
+This might be achieved by obtaining configuration at startup,
+before starting to load any content.
+
+If the necessary configuration cannot be obtained,
+or is obviously stale,
+an implementation can provide randomized responses
+that are indistinguishable from a real response
+when <a for=PrivateAttribution>measureConversion()</a> is invoked.
```

Comment:
This might not work for things like key identifiers.  We need to double check that for the different aggregation services.